src/mqt/yaqs/core/data_structures/simulation_parameters.py

```
@@ -225,6 +227,7 @@ def __init__(
             Number of threads to use for single-trajectory simulations (BLAS/LAPACK).
             Defaults to 1 for efficiency on small/medium bond dimensions.
         """
+        self.noise_model: NoiseModel | None = None
```

Ruff (0.15.0)

[error] Undefined name `NoiseModel`

(F821)